dll/win32/shell32/dialogs/dialogs.cpp

```diff
@@ -37,9 +37,10 @@ typedef struct
 {
     BOOL bFriendlyUI;
     BOOL bIsButtonHot[2];
-    HBITMAP hImageStrip;
-    HBRUSH hBrush;
+    HBITMAP hBitmap[3];
+    HTHEME hTheme[5];
     HFONT hfFont;
+    WCHAR StaticTexts[3][128];
```

Comment:
Is the "3" like NUMBER_OF_BUTTONS ?

Reply:
@HBelusca Should have been `NUMBER_OF_BUTTONS + 1` since it accommodates for buttons labelling as well as title box of the dialog box.
I will make a macro for it to remove the magic numbers.